### PR TITLE
Navigation and Layout fixes 

### PR DESCRIPTION
**Pull request by adriencyberspace**
# Pull Request Template

## Description

This PR addresses multiple navigation and layout fixes:
1. Top level and single level navigation items are styled the same
2. Max width of navigation bar is the same max width as footer
3. Removed Error and loading state on navbar
4. Adds secondary navigation bar to missing pages
5. Adds a max width to secondary navigation
6. Adds a max width the search results pages to not be wider than navigation width
7. Adds footer to all pages and removes individually placed footers from site
8. Some file renaming (I will keep this to a minimum moving forward)

## How Can This Be Tested/Reviewed?

Provide instructions so we can review.

Describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have added QA notes to the issue with applicable URLs
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [x] I have reviewed the changes in a mobile view
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have assigned reviewers
- [x] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

### CODE REVIEW

File: app/components/ui/Navigation.tsx [hunk 1]
```diff
@@ -34,8 +34,10 @@ const BURGER_STYLES = {
   },
 };
 
+// TODO: use @headlessui/react components for a11y
```
Comment (seanmalbert) [inline app/components/ui/Navigation.tsx:37]: we should be able to bring over what we did for Bookshare, at least in part, no?

File: app/components/ui/Navigation.tsx [hunk 2]
```diff
@@ -144,7 +136,7 @@ export const Navigation = () => {
               <div className={styles.navigationMenuTranslate}>
                 <GoogleTranslate />
               </div>
```
Comment (seanmalbert) [inline app/components/ui/Navigation.tsx:138]: I don't think it needs to be done here, or maybe not at all, but the nav section might load more cleanly if there's space allocated for translate before it loads, so users don't see the whole nav shift to the left once the translation dropdown loads.

**Review by seanmalbert — APPROVED** [review]
Looks good, @adriencyberspace ! Left a couple of comments, but nothing blocking.

File: app/components/ui/Navigation.tsx (hunk 1 above)
Comment (adriencyberspace) [reply app/components/ui/Navigation.tsx:37]: Yeah, I wish we wouldn't have done this fully custom

File: app/components/ui/Navigation.tsx (hunk 2 above)
Comment (adriencyberspace) [reply app/components/ui/Navigation.tsx:138]: Added this in!

**Pull request merged**